src/tiled/tileseteditor.cpp

```diff
@@ -826,6 +861,40 @@ void TilesetEditor::removeTerrainType()
         undoStack->endMacro();
 }
 
+void TilesetEditor::currentWangSetChanged(const WangSet *wangSet)
+{
+    TilesetView *view = currentTilesetView();
+    if (!view)
+        return;
+
+    if (wangSet)
+        view->setWangSet(wangSet);
+}
+
+void TilesetEditor::addWangSet()
+{
+    Tileset *tileset = currentTileset();
+    if(!tileset)
+        return;
+
+    //2 and 0 are default values for number of edges and corners TODO define this some where better?
```

Comment:
This is fine for now, though we should probably add a "NewWangSetDialog" class so that the user can set up his wang set upon creation (where there would be space to explain the corner/edges count a bit and show the presets).